services/spar/test-integration/Test/Spar/APISpec.hs

```diff
@@ -849,7 +851,7 @@ specCRUDIdentityProvider = do
         it "responds with 2xx; makes IdP available for GET /identity-providers/" $ do
           env <- ask
           (owner, _) <- call $ createUserWithTeam (env ^. teBrig) (env ^. teGalley)
-          metadata <- Aeson.encode . (IdPMetadataValue mempty) . fst <$> makeTestIdPMetadata
+          (SampleIdP (Aeson.encode . (IdPMetadataValue mempty) -> metadata) _ _ _) <- makeSampleIdPMetadata
```

Comment:
Using the field accessor could avoid view patterns.

Reply:
b97a104